src/markdown-pages/explore-docs/nerdstoragevault.mdx

```diff
@@ -0,0 +1,142 @@
+---
+path: '/explore-docs/nerdstoragevault'
+title: 'Intro to NerdStorageVault'
+duration: '30 min'
+template: 'GuideTemplate'
+description: 'Intro to NerdStorageVault on New Relic One'
+tileShorthand:
+  title: 'Intro to NerdStorageVault'
+  description: 'Learn about NerdStorageVault usage'
+tags:
+  - nerdstoragevault
+  - nerdstoragevault components
+  - new relic one apps
+  - data access
+  - encrypted storage
+resources:
+  - title: 'Build an app'
+    url: /build-apps
+  - title: 'Intro to New Relic One API components'
+    url: /explore-docs/intro-to-sdk
+---
+
+<Intro>
+
+To help you build a [New Relic One application](/use-cases/build-new-relic-one-applications),
+we provide you with the [New Relic One SDK](/explore-docs/intro-to-sdk).
+On this page, you’ll learn how to use NerdStorageVault to store data in an encrypted
+storage solution.
+
+</Intro>
+
+## Using NerdStorageVault in your apps
+
+NerdStorageVault is used to store and retrieve sensitive secrets data such as:
+
+- personal access tokens
+- license keys
+- API keys
+- other third party secrets
+
+<Tip>
+
+NerdStorageVault is encrypted with AES-256 encryption
+
+</Tip>
+
+## Usage considerations
+
+- This storage is unique per Nerdpack, and can't be shared with any other Nerdpack.
+- NerdStorageVault can currently only be used within the context of a Nerdpack.
+- You cannot make a direct query or mutation to NerdStorageVault.
+- You can explore the fields and response types via the [NerdGraph API explorer](https://api.newrelic.com/graphiql).
+- The only supported scope is ACTOR, or data that relates to a particular user.
+
+<Tip>
+
+A trusted nerdpack ID header value is provided by service gateway when requests come from nerdlets and
```

Comment:
Another question on this, sorry 😂 , do we talk about service gateway externally? not sure if users know what that is or not

Reply:
Oh if that's an internal term  only I can take it out.